### ISSUE TITLE
SimCLR Imagenet Results Reproduction

### ISSUE THREAD

**Issue by zlapp** [PyTorchLightning/pytorch-lightning-bolts#152]
Hi,

Firstly thank you for this very cool repository.
I would like to use your implementation of SimCLR to reproduce the Imagenet results. 

I saw the following command in the implementation:
            # imagenet
            python simclr_module.py
                --gpus 8
                --dataset imagenet2012
                --data_dir /path/to/imagenet/
                --meta_dir /path/to/folder/with/meta.bin/
                --batch_size 32

Does this command match the papers results? How long does the self-supervised training take? Are there any stats that could be added for reference (hardware used, times ect.).

Thanks :)

**Comment by ananyahjha93** [comment]
@zlapp we are working on a version that replicates the paper results. However, the command mentioned above will not replicate imagenet results for simclr. You'll need a very large batch size (and hence more GPUs) to provide negative samples for the contrastive loss to work. This command is just an example of running SimCLR on other datasets, once we have verified the official results.
